package-res/ccc/core/base/axis/axis.js

```diff
@@ -28,6 +28,13 @@ def('pvc.visual.Axis', pvc.visual.OptionsBase.extend({
 
         // Fills #axisIndex and #typeIndex
         chart._addAxis(this);
+
+        // CDF603 C
+        // uses the state of the axis to update its objects
+        // elements of state will be elements of axis
+        
+        if(keyArgs && keyArgs.state) $.extend(this, keyArgs.state); 
```

Comment:
See comment below relating use of jQuery.
Also, shouldn't `this._state` be being updated, instead of `this`?
You could initialize `this._state` to an empty object and  then do `this.setState(keyArgs.state);`.